From 4685b99a12e20172b24f11357a9fdd5b8a81bf59 Mon Sep 17 00:00:00 2001
From: Subrata Chatterjee <schatterjee@habana.ai>
Date: Fri, 10 Jun 2022 12:48:22 +0300
Subject: [PATCH] [SW-70527] gaudi2: increase ppboot size to 20kb

As sram bist code increases ppboot size, so increased
the size further from 19KB to 20KB.

Signed-off-by: Subrata Chatterjee <schatterjee@habana.ai>
Change-Id: I9e5aadd2837e7949d035155aed74c99f67a523eb
---

diff --git a/arch/gaudi2/arc/Makefile b/arch/gaudi2/arc/Makefile
index e8eb7d5..805c564 100644
--- a/arch/gaudi2/arc/Makefile
+++ b/arch/gaudi2/arc/Makefile
@@ -1,7 +1,7 @@
 include $(HL_DEV_ARCH_COMMON)/Makefile
 
 FW_SIZE := 0x8000
-PPBOOT_MAX_SIZE := 19456
+PPBOOT_MAX_SIZE := 20480
 PREBOOT_BIN_MAX_SIZE := 130048
 
 LDFLAGS += -Wl,--just-symbols $(HL_DEV_ARCH)/exports.elf
diff --git a/arch/gaudi2/arc/common.h b/arch/gaudi2/arc/common.h
index 85d61b8..bbad465 100644
--- a/arch/gaudi2/arc/common.h
+++ b/arch/gaudi2/arc/common.h
@@ -9,7 +9,7 @@
 #include <bitops.h>
 #include <sizes.h>
 
-#define PPBOOT_SIZE			0x4C00		/* 19KB */
+#define PPBOOT_SIZE			0x5000		/* 20KB */
 #define PREBOOT_SIZE			0x1FC00		/* 127KB */
 
 #define FLASH_EFUSE_OFFSET		(FLASH_SECONDARY_OFFSET + \
